File: app/src/main/java/com/example/phonebook/model/db/UsersDB.java
package com.example.phonebook.model.db;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.example.phonebook.model.dao.usersRepoDAO;
import com.example.phonebook.model.data.User;

@Database(entities = {User.class}, version = 1, exportSchema = false)
public abstract class UsersDB extends RoomDatabase {
    public abstract usersRepoDAO getUsersDB();
}

File: app/src/main/java/com/example/phonebook/model/db/UsersDB.kt
package com.example.phonebook.model.db

import androidx.room.Database
import androidx.room.RoomDatabase

import com.example.phonebook.model.dao.UsersRepoDAO
import com.example.phonebook.model.data.User

@Database(entities = [User::class], version = 1, exportSchema = false)
abstract class UsersDB : RoomDatabase() {
    abstract fun getUsersDB(): UsersRepoDAO
}
